pkg/glusterfs/api/types.go

```diff
@@ -693,3 +694,53 @@ func ValidateIds(v interface{}) error {
 	}
 	return nil
 }
+
+type NodeHealthStatus struct {
+	NodeId     string    `json:"node"`
+	Host       string    `json:"host"`
+	Up         bool      `json:"online"`
+	LastUpdate time.Time `json:"updated"`
+}
+
+type DbState uint8
```

Comment:
I don't think this kind of detail is desirable in the api. Just have an enumerated string type with the ok/faulty info needed. That seems where you were going with the Healthz type, so I suggest simply removing this type from the api, and if you still think you want these types, locate them inside the app/glusterfs (for example).
The important part is not to put anything extra in the API.

Reply:
removed